frontend/test/metabase/scenarios/custom-column/cc-help-text.cy.spec.js

```diff
@@ -34,7 +34,7 @@ describe("scenarios > question > custom column > help text", () => {
     cy.findByText("round([Temperature])");
 
     // Click outside of formula field instead of blur
-    cy.findByText(/Field formula/i).click();
+    cy.findByText(/Expression/i).click();
```

Comment:
would it be possible to find this element using a simple string? [Ref](https://www.notion.so/metabase/43-Update-Style-Guide-2bcbc77ff4f34c9bb74a6623db22c9d6?pvs=4#9d35175600c549cc91f8bfcce5bdafd3)